### DISCUSSION TITLE
Resolve invalid dependencies when upgrading to React18 with react-select v5.7.0

### DISCUSSION THREAD

**Discussion by seanRoo** [JedWatson/react-select#5578] [Ideas]
We are migrating to React v18 and we are seeing invalid nested dependencies in the `react-select` dependency tree when running `npm ls react`.
We have a work around by using "overrides" in our `package.json`, forcing the invalid packages to use React v18, but it would be great if this was resolved so we didn't have to do that.
![Screen Shot 2023-03-09 at 1 18 44 PM](https://user-images.githubusercontent.com/33811391/224162960-9117e67a-e9f3-47c9-a0dd-ac5269f074b3.png)

**Comment by Methuselah96** [comment]
This is an issue with `react-inspector` and `react-element-to-jsx-string` not supporting React 18, neither of which are dependencies of react-select, they're coming from some other dependency in your project (likely Storybook).

**Reply by seanRoo** [reply]
Thanks for confirming this @Methuselah96 👍
